recipes/websocketpp/all/conanfile.py

```diff
@@ -1,35 +1,42 @@
 import os
 from conans import ConanFile, tools
 
+required_conan_version = ">=1.33.0"
+
 
 class WebsocketPPConan(ConanFile):
     name = "websocketpp"
     description = "Header only C++ library that implements RFC6455 The WebSocket Protocol"
-    topics = ("conan", "websocketpp", "websocket", "network", "web", "rfc6455", "header-only")
+    topics = ("websocketpp", "websocket", "network", "web", "rfc6455", "header-only")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/zaphoyd/websocketpp"
     license = "BSD-3-Clause"
     settings = "os", "arch", "compiler", "build_type"
     exports_sources = ["patches/*"]
-    options = {"asio": ["boost", "standalone"]}
-    default_options = {"asio": "boost"}
-    
+    options = {"asio": ["boost", "standalone", "none"],
+               "openssl": [True, False],
+               "zlib": [True, False]}
+    default_options = {"asio": "boost", "openssl": True, "zlib": True}
```

Comment:
Using a `with_` prefix is used by a few other recipes
I'm also unsure how to name the option value to disable asio. We normally use python `None`. `False` would be fine too.
```suggestion
    options = {
        "asio": ["boost", "standalone", False],
        "with_openssl": [True, False],
        "with_zlib": [True, False],
    }
    default_options = {
        "asio": "boost",
        "with_openssl": True,
        "with_zlib": True,
    }
```